src/Imdb/Person.php

```diff
@@ -1085,6 +1085,14 @@ protected function getPage($page = null)
 
         return $this->page[$page];
     }
-
+    
+    // this will, in case of a redirect by imdb site) get the new person id
```

Comment:
Could you document the return type here to make it very clear it's a stringy number and not 'nm00000', and put it in a proper docbloc.
The equivalent method for titles looks like this:
```php
/**
     * Get the ID for the title we're using. There might have been a redirect from the ID given in the constructor
     * @return string|null e.g. 0133093
     */
```

This method is public and needs moving above the protected methods please

Reply:
The `/** @return` stuff is phpdoc, and has a specific syntax that makes it useful for IDEs and other things that want to do static analysis (we run phpstan to check the code)
https://docs.phpdoc.org/2.9/references/phpdoc/tags/return.html
As it says on that page you can't have two `@return` tags. It's fairly bad form to return more than one type, but you can get away with null and you can document as many as you want with the union operator (`string|null|whatever`)

Reply:
By test I mean a (unit|integration) test in https://github.com/tboothman/imdbphp/blob/master/tests/PersonTest.php
Because every method in this library just scrapes html that's not under my control it can change at any time. The tests are useful for noticing when stuff stops working (sometimes subtly) and for figure out exactly what it was meant to do in certain circumstances.

Tests run daily btw, you can see them here https://github.com/tboothman/imdbphp/actions

A pretty related method to this stopped working a while ago because imdb changed the status code they used for redirects. You can see it failing here:
https://github.com/tboothman/imdbphp/runs/5339015362?check_suite_focus=true
Likely would've remained broken for a long time before someone noticed but the test showed it up and gave a clear reason why.

Reply:
Okay, so the whole if else construction that i made is not needed because the function will always return null if the first condition is not met. I will remove it again, i didn't know that.
i will adjust the phpdoc again, didn't know that either about returns
ah okay i will try to come up with a test